### ISSUE TITLE
I made a hairpin for monika

### ISSUE THREAD

**Issue by Sujeito17** [Monika-After-Story/MonikaModDev#2404]
I'm bad at drawing so this is just a test, I did my best to look beautiful
![test](https://user-images.githubusercontent.com/41938650/45591260-83ce0880-b945-11e8-84e3-0385eabc7b71.png)In Monika's hair
Original ![oie_transparent 2](https://user-images.githubusercontent.com/41938650/45591261-892b5300-b945-11e8-8274-b97f795d06c1.png)
Maybe I'll make one of Monika's ... As long as you wait, you're going to have to keep the "Subject 17" version XD

**Comment by Sujeito17** [comment]
Monika version![torso-down](https://user-images.githubusercontent.com/41938650/45591586-1aeb8e00-b94f-11e8-9b8f-8f1d5b6f68ed.png)In Monika's hair

![monika_lit__club_____danganronpa_8_bit_style_by_tomaswolfy15-dcg30p8](https://user-images.githubusercontent.com/41938650/45591591-3b1b4d00-b94f-11e8-9a1f-59da0fa6fcb2.png)Original
Was kind of hard to do the Monika version, I still think the "Subject 17" version is better, I left the pixel arts if someone wants to improve.

**Comment by SaiPrithvi123** [comment]
"Subject 17"? Isn't he Desmond Miles?

**Comment by Sujeito17** [comment]
That's right, well "Subject 17" is the translation from Portuguese to English, when I chose that name I was a fan of Assassin's Creed (I still am), but the name has nothing to do with Assassin's Creed anymore and I created my own story of why I call myself that.
(It's not just a story, it's several stories)
(I do not know why, but I'd rather be called Subject 17 than to call me by the real name)

**Comment by Rai99** [comment]
This would probably go on the community branch. 

I think it's great that you wanted to make Monika a hairpin (and make Monika _into_ a hairpin), but I think in order to get any real consideration, it would have to be a generic hairpin. Personally, I have no clue what that first one is from, and whatever it`s from I don't even know if I or Monika would be a big enough fan to get a hairpin of it. 

Also, while pixel art might work, I think we would aim for a more detailed hairpin for Monika. 

Finally, does Monika really need a hairpin or would the option to let her have one be a community branch thing?

**Comment by Sujeito17** [comment]
@Rai99 (The first is a pixel art that I created myself is not inspired by anything)
I can not draw well if I knew I would have made a hairpin that was not pixel art and I would not stop there, I would have done a lot of things for Monika ;-; (that's why I left the pixel arts, should have people who know how to make better pixel arts than I do)
Well it's just an accessory I do not think it's a very relevant thing for now

**Comment by OrangTea** [comment]
Instead of a character, what if it was a monitor, to represent the 4th wall breaks of Monika? Like this... ![hairpin](https://user-images.githubusercontent.com/43391253/45720133-edbdfc00-bb57-11e8-8b29-f55d126b3533.png)

**Comment by Sujeito17** [comment]
@OrangTea It's good, how many more hair clips the better

**Comment by OrangTea** [comment]
@Sujeito17 I just made 4 new ones! Let me know if you have any ideas!
The DDLC logo:
![torso-down-hp1](https://user-images.githubusercontent.com/43391253/45724447-169bbc80-bb6b-11e8-8daf-d2bb6c5cf2df.png)
A green music note:
![torso-down-hp2](https://user-images.githubusercontent.com/43391253/45724460-30d59a80-bb6b-11e8-927b-c28ea919bb4e.png)
Yuri's Hairclip:
![torso-down-hp3](https://user-images.githubusercontent.com/43391253/45724774-852d4a00-bb6c-11e8-8e25-b9b37371c4b8.png)
Monika's Pen:
![torso-down-hp4](https://user-images.githubusercontent.com/43391253/45724779-8a8a9480-bb6c-11e8-9ad7-88b66be8edf4.png)

**Comment by ThePotatoGuy** [comment]
#2433 ,closing until we look into this

**Comment by ThePotatoGuy** [comment]
[reopening as this can be done now]

**Comment by ThePotatoGuy** [comment]
linking #3174 

since it is also not full layers, will need to be moved in correct spot
